backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -135,4 +135,22 @@ Java_backtraceio_library_BacktraceDatabase_disable(JNIEnv *env, jobject thiz) {
     Disable();
 }
 
+extern "C"
+JNIEXPORT jboolean JNICALL
+Java_backtraceio_library_base_BacktraceBase_EnableCrashLoopDetectionBacktrace(JNIEnv *env, jclass clazz) {
```

Comment:
This shouldn't belong to the BacktraceBase class - instead, it should belong to BacktraceDatabase. Why? To enable and do any operation via Crashpad we need a database. If we do not have the database, this code should fail. There is no reason to run it in the client if the database is unavailable. It belongs to the database domain

Reply:
Moved to BacktraceDatabase